[2024-01-23T18:14:39.969+0200] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgres_to_redshift.load_to_redshift_table manual__2024-01-23T16:14:31.436463+00:00 [queued]>
[2024-01-23T18:14:40.048+0200] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgres_to_redshift.load_to_redshift_table manual__2024-01-23T16:14:31.436463+00:00 [queued]>
[2024-01-23T18:14:40.048+0200] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-23T18:14:40.061+0200] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): load_to_redshift_table> on 2024-01-23 16:14:31.436463+00:00
[2024-01-23T18:14:40.063+0200] {standard_task_runner.py:60} INFO - Started process 54776 to run task
[2024-01-23T18:14:40.065+0200] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'postgres_to_redshift', 'load_to_redshift_table', 'manual__2024-01-23T16:14:31.436463+00:00', '--job-id', '295', '--raw', '--subdir', 'DAGS_FOLDER/postgres_to_redshift.py', '--cfg-path', '/tmp/tmp6pyrg9dg']
[2024-01-23T18:14:40.067+0200] {standard_task_runner.py:88} INFO - Job 295: Subtask load_to_redshift_table
[2024-01-23T18:14:40.094+0200] {task_command.py:423} INFO - Running <TaskInstance: postgres_to_redshift.load_to_redshift_table manual__2024-01-23T16:14:31.436463+00:00 [running]> on host DESKTOP-9NHC01K.
[2024-01-23T18:14:40.152+0200] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='postgres_to_redshift' AIRFLOW_CTX_TASK_ID='load_to_redshift_table' AIRFLOW_CTX_EXECUTION_DATE='2024-01-23T16:14:31.436463+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-23T16:14:31.436463+00:00'
[2024-01-23T18:14:40.163+0200] {base.py:83} INFO - Using connection ID 'my_aws_cnn' for task execution.
[2024-01-23T18:14:40.164+0200] {connection_wrapper.py:378} INFO - AWS Connection (conn_id='my_aws_cnn', conn_type='aws') credentials retrieved from login and password.
[2024-01-23T18:14:43.473+0200] {python.py:201} INFO - Done. Returned value was: None
[2024-01-23T18:14:43.487+0200] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=postgres_to_redshift, task_id=load_to_redshift_table, execution_date=20240123T161431, start_date=20240123T161439, end_date=20240123T161443
[2024-01-23T18:14:43.543+0200] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-23T18:14:43.554+0200] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
